components/energylevel.css

```
@@ -0,0 +1,147 @@
+.energylevel {
+  width: 315px;
+  height: 208px;
+  display: grid;
+  grid-template-columns: 32px repeat(5, minmax(32px, 1fr));
+  grid-template-rows: repeat(6, minmax(0, auto));
+  grid-template-areas:
+    'percentage  bars   bars   bars   bars   bars'
+    'percentage  bars   bars   bars   bars   bars'
+    'percentage  bars   bars   bars   bars   bars'
+    'percentage  bars   bars   bars   bars   bars'
+    'percentage  bars   bars   bars   bars   bars'
+    '.           time time time time time';
+}
+
+.energylevel h3 {
```

Why not give this heading a class name to be more specific?
```suggestion
.energylevel__heading {
```